client/src/components/Navbar/Navbar.tsx

```diff
@@ -22,9 +22,9 @@ const Navbar = (): JSX.Element => {
 
           <Grid item container alignItems="center" xs={4}>
             <Router>
-              <Grid item container justifyContent="space-evenly" alignItems="center">
+              <Grid item container style={{justifyContent:"space-evenly", alignItems:"center"}}>
```

Comment:
Generally it's preferred to use mui props over custom styling - any particular reason for this change (as well as the other instances of the same change)? 

NOTE: You can inspect the props for each component in mui at its API page (e.g. for `Grid` see [here](https://v4.mui.com/api/grid/#props))